CHANGES.txt

```diff
@@ -1,3 +1,5 @@
+Fix race condition bug during local session repair (CASSANDRA-17335)
```

Comment:
This entry should go nder the 4.0.4 section

Reply:
Moving it